panel/src/components/Forms/Writer/Writer.vue

```diff
@@ -450,6 +452,14 @@ export default {
 	font-family: var(--font-mono);
 }
 
+.k-writer blockquote {
+	font-size: var(--text-xl);
```

Comment:
The xl font size looks too big because fonts are shrunk in the new UI. `lg` is better.

```suggestion
	font-size: var(--text-lg);
```